src/test/java/edu/kit/provideq/toolbox/format/GmlTests.java

```diff
@@ -0,0 +1,104 @@
+package edu.kit.provideq.toolbox.format;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import edu.kit.provideq.toolbox.exception.ConversionException;
+import edu.kit.provideq.toolbox.format.gml.Gml;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class GmlTests {
+  @ParameterizedTest
+  @MethodSource("testToStringParameters")
+  public void testToString(String gmlString) throws ConversionException {
+    gmlString = gmlString.replace("\n", System.lineSeparator());
```

Comment:
This should be CRLF too, as discussed in comment above

Reply:
Yes, that's what this replace part is for. It converts the gmlString from the string block (which always uses LF-only) to the CRLF format.

Reply:
But isn't `System.lineSeparator()` systemdependent?

Reply:
System.lineSeparator() is what is used anywhere else. This means the server is consistent in itself so for that reason there shouldn't be an issue. However, servers deployed on different machines might result in different results, so we might want to standardize this to one specific line separator.